client/signup/jetpack-connect/index.jsx

```diff
@@ -17,7 +17,7 @@ import Main from 'components/main';
 import JetpackConnectNotices from './jetpack-connect-notices';
 import SiteURLInput from './site-url-input';
 import { getSiteByUrl } from 'state/sites/selectors';
-import { requestSites } from 'state/sites/actions';
+import QuerySites from 'components/data/query-sites';
```

Comment:
hmm, why not? internally it's doing the same that we were doing in this component. And in this use-case, we are not using the computed attributes at all, it's just being used to check if the site is there or not